webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -1,7 +1,21 @@
 import $ from 'jquery';
+import { push } from 'connected-react-router';
 import store from './react_app/redux';
 import * as LayoutActions from './react_app/components/Layout/LayoutActions';
 
+/**
+ * Build a url with query params
+ * @param {String} url - the base url i.e `/hosts`
+ * @param {String} searchQuery - the query params, i.e [['per_page': 4], ['page': 2]]
+ */
+export const pushUrl = (url, queryParams) => {
+  const searchParams = new URLSearchParams('');
```

Comment:
Should we start using https://github.com/sindresorhus/query-string instead?

Reply:
`URLSearchParams` works as expected, no?